src/testdir/term_util.vim

```diff
@@ -122,7 +122,6 @@ func RunVimInTerminal(arguments, options)
     if term_getsize(buf) != [rows, cols]
       call term_setsize(buf, rows, cols)
     endif
-    call assert_equal([rows, cols], term_getsize(buf))
```

Comment:
Why is this assert removed?

Reply:
I had added a _lot_ more asserts here and got overzealous as I rewound those. I'll add it back.

Reply:
For what it's worth, I suspect there is a bug that makes this assertion execute slightly early in the GUI tests. I was seeing failures of tests that used screendumps on vim sessions run in a terminal where the failed dump was 1 column too narrow while this assertion was evaluating true. I added some judicious `sleep` calls so I could observe and capture a screenshot with Xnest. I've overlaid them here. You can see the GUI shell is the same width in each, but for some reason the terminal actually acquires 1 fewer columns.

![vim_term_size_offset2](https://github.com/user-attachments/assets/23192536-9d2c-494e-bd51-4623164cfc22)